services/apps/alcs/src/alcs/search/planning-review/planning-review-advanced-search.service.ts

```diff
@@ -100,6 +100,8 @@ export class PlanningReviewAdvancedSearchService {
     query = this.compileSearchByNameQuery(searchDto, query);
     query = this.compileDecisionSearchQuery(searchDto, query);
     query = this.compileDateRangeSearchQuery(searchDto, query);
+    query = this.compileDateRangeSearchQuery(searchDto, query);
```

Comment:
This looks like an exact copy of the line above? Is this intentional?

Reply:
good catch, probably a merge conflict, fixed